### ISSUE TITLE
Menu Navigation gets stuck and says "blank" for some users

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32414]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

This issue does not seem to affect all users but I can reproduce it.  Could it have anything to do with monitor size or screen resolution as I have a very old monitor and it only happens to some others if they aren't maximised.

Steps:
1.  Press Alt F to open the file menu.  
2. Press right arrow 3 times and hear screen reader announce. "Edit panel, View panel, Add 'panel" along with whatever the top available item is (this is expected).
3. Press Right arrow again.  I hear blank and get this on all subsequent presses of right arrow, suggesting navigation is stuck.
4. Use left arrow to navigate back through each of the menus and all work as expected.
5. With no score open press Alt E. As expected you hear Edit panel, preferences.
6. Try moving with either up or down arrow and the screen reader says "blank".  Presumably this is because there is no where to move and it says blank when it can't navigate.

 

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
